.github/workflows/header-gen.yml

```diff
@@ -4,12 +4,14 @@ on:
   release:
     types: [created]
   push:
-    branches: '*'
```

Comment:
VS code flagged this syntax as invalid for me. Though I suspect it may work fine it doesn't seem worth arguing!

Reply:
Interesting! I think I've used the same syntax in other workflow files, I'll keep it in mind the next time I'll update them 👍